### PR TITLE
Handle signals properly in FUSE container

### LINKED ISSUE #9840: Update FUSE integration scripts to be able to be launched in the foreground
**Is your feature request related to a problem? Please describe.**

Recent the `./bin/launch-process` script was added to launch Alluxio processes in the foreground. This is now being used by `alluxio-start.sh` and the docker container entrypoint so that containerized Alluxio processes behave properly when signals are sent to the container.

However, the FUSE daemon launched from the script `./integration/fuse/bin` directory doesn't launch the process in the foreground, meaning that our containerized Alluxio FUSE daemon isn't a good citizen in the container world.

We should update the docker entrypoint and a`alluxio-fuse` script so that it can handle signals and terminate cleanly in docker or K8s.

**Describe the solution you'd like**

Similar to `launch-process`, we'll need to use `exec` and limit use of `nohup` within the FUSE daemon script.

**Describe alternatives you've considered**

We could write a separate entrypoint for the fuse container, but it would still require changes to the fuse daemon script.

**Urgency**

medium

**Additional context**

N/A

### PR DESCRIPTION
**Pull request by ZacBlanco**
Resolves #9840

### CODE REVIEW

File: integration/docker/Dockerfile.fuse [hunk 1]
```diff
@@ -21,7 +21,7 @@ RUN apt-get update && apt-get install -y --no-install-recommends software-proper
     add-apt-repository -y ppa:openjdk-r/ppa && \
 	apt-get update && \
 	apt-get install -y --no-install-recommends \
-	openjdk-8-jdk openjdk-8-jre-headless libfuse-dev && \
+	openjdk-8-jdk openjdk-8-jre-headless fuse && \
```
Comment (madanadit) [inline integration/docker/Dockerfile.fuse]: why is this change needed?

**Comment by LuQQiu** [comment]
Docker and kubernetes require all processes running as foreground processes?

**Comment by madanadit** [comment]
@LuQQiu The proper way in Kubernetes is to run processes in the foreground. Multiple processes are run in separate containers (potentially part of the same pod = the schedulable unit)

**Comment by LuQQiu** [comment]
@madanadit  Got it! Thanks for the details!

File: integration/docker/Dockerfile.fuse [hunk 2]
```diff
@@ -50,6 +50,8 @@ RUN if [ ${ALLUXIO_USERNAME} != "root" ] \
       mkdir -p /journal && \
       chown -R ${ALLUXIO_UID}:${ALLUXIO_GID} /opt/* /journal && \
       chmod -R g=u /opt/* /journal; \
+      mkdir /alluxio-fuse && \
```
Comment (LuQQiu) [inline integration/docker/Dockerfile.fuse]: I am still curious why previously docker and kubernetes can launch Alluxio fuse containers without creating the alluxio-fuse folder?

Comment (ZacBlanco) [reply integration/docker/Dockerfile.fuse]: Previously this worked because the container ran as `root` (uid 0), so the directory could be created at runtime. Now, because we're running as `alluxio`, permissions to `/` are denied. So we create the folder and `chown` at container creation time

**Comment by ZacBlanco** [comment]
FYI I tested this locally and everything seemed to work. Only issue was that my master container ran as root instead of Alluxio. Due to the mismatch in users. I had to `chown` the local UFS volume as `alluxio` to allow files to be created on the fuse mount.

**Comment by ZacBlanco** [comment]
alluxio-bot, merge this please
